search/scorer/scorer_knn.go

```diff
@@ -33,28 +34,25 @@ func init() {
 }
 
 type KNNQueryScorer struct {
-	queryVector      []float32
-	queryField       string
-	queryWeight      float64
-	queryBoost       float64
-	queryNorm        float64
-	docTerm          uint64
-	docTotal         uint64
-	options          search.SearcherOptions
-	includeScore     bool
-	similarityMetric string
+	queryVector            []float32
+	queryField             string
+	queryWeight            float64
```

Comment:
How large can these value be queryWeight, queryBoost, queryNorm ?

Instead of float64, can we use something smaller ?

Reply:
nvm, this was unrelated to the change.
But, we can still think about it :)

Reply:
Yep, mainly used these since term scorer too used the same.